seadssite/forms.py

```diff
@@ -33,11 +33,11 @@ class UserForm(forms.ModelForm):
     )
     username = forms.CharField(min_length=4, max_length=20, required=True)
     email = forms.EmailField(min_length=4, max_length=20, required=True)
-    phone = forms.CharField(min_length=7, required=True,
+    phone = forms.CharField(required=True,
         validators=[
             RegexValidator(
-                regex='\(?([0-9][0-9][0-9])?\)?[0-9][0-9][0-9]\-?[0-9][0-9][0-9][0-9]$',
-                message='Please enter a valid phone number.',
+                regex='^[0-9]{10}$',
+                message='Please enter a 10 digit phone number with no punctuation.',
```

Comment:
I think this is an improvement. Maybe we can replace this with or add a check with a messaging service?